newdle/client/src/components/Navigator.module.scss

```diff
@@ -22,15 +23,5 @@
 }
 
 .step-body {
-  background: white;
-  margin: 1rem 0em;
-  padding: 1em 1em;
-
-  .title {
-    color: $pink;
-  }
-
-  .description {
```

Comment:
You are still using `styles['description']` in `Navigator.js`

Reply:
You're absolutely right, thanks! 👍